### ISSUE TITLE
Unit 3 Lab 2

### ISSUE THREAD

**Issue by veldyn** [veldyn/b2c-sysadmin-2024#11]
For Unit 3 Lab 2 we want to walkthrough the process of creating an Nginx server in Docker and deploying an instance. This will cover stateless containers and provide a the first practical use-case for Docker to the students that they built with the aid of the walkthrough.

**Comment by bgoss07** [comment]
Bethany to make /usr/share/nginx directory on Wed to see if that will override default nginx image in docker.

**Comment by veldyn** [comment]
Today we worked on installing Docker on our machines and configuring Nginx. We're having trouble finding where the configuration files are for Nginx, since they are not in the default directories. We don't want to install Nginx, so we're going to investigate more next time into where the source files might be.

We finished up the introduction to U3L2, and completely wrapped up everything from U2L1 and added to the Git.

**Comment by bgoss07** [comment]
02/12 Today we added screenshots to our instructions for custom Docker container & made a set of questions for output. This finished U3L2 for now.

**Comment by veldyn** [comment]
Today we put the wraps on Unit 3 Lab 2 with hosting the Nginx server. We ensured all the commands worked properly and finished grabbing the required screenshots for the lab. We added a questions portion at the end to test student comprehension, and requested some screenshots of a running instance of Nginx hosted on their local server with a specific key-phrase to ensure the lab was actually completed.

We've uploaded the Lab onto the GitHub page and are now focused on Lab3. We're thinking we'll do a database, since we want Lab3 to be an example of a stateful docker container. MongoDB is what we've decided on for now, since it's commonly used in web applications and we have more to learn personally from it than something like MySQL.

We expect development of this lab to take a bit longer than Nginx, since we will have to demonstrate the lab being stateful, as well as our lack of experience in this domain.
